sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/planner/BeamRuleSets.java

```diff
@@ -133,7 +134,7 @@
           // remove unnecessary sort rule
           // https://issues.apache.org/jira/browse/BEAM-5073
```

Comment:
This is now closed. Maybe it is worth cleaning up the comment now?

Reply:
Nice catch. I decided to re-open that Jira (so keep the work for re-enabling SortRemoveRule). 

Keep comments as what it is now.